backend/src/main/java/vangoh74/backend/model/RoundState.java

```diff
@@ -0,0 +1,9 @@
+package vangoh74.backend.model;
+
+public enum RoundState {
+
+    PRE_FLOP,
+    PRE_TURN,
+    PRE_RIVER
+
```

Comment:
Spannend. Und wenn der River gelegt ist, dann ist der RoundState leer und das Programm weiß so, dass der letzte State in dieser Runde ist?

Reply:
Hi Marlin! Genau! Wobei, wo Du jetzt nachfragst, ich sehe dass es bevor der Showdown kommt, noch eine Pre-Schowdown Wettrunde gibt... dieser State werde ich gleich einbauen! :-) Danke!